templates/home/home-fr.html

```diff
@@ -63,46 +63,25 @@ <h3><a href="https://www.canada.ca/fr/services/entreprises.html">Entreprises et
-				<h3><a href="https://www.canada.ca/fr/services/sante.html">Santé</a></h3>
-				<p>Alimentation, nutrition, maladies, vaccins, médicaments, sécurité des produits et rappels</p>
-			</div>
 			<div class="col-lg-4 col-md-6">
 				<h3><a href="https://www.canada.ca/fr/services/impots.html">Impôts</a></h3>
 				<p>Impôt sur le revenu, TPS/TVH, limites de contribution, credits d’impôt, organismes de charité </p>
 			</div>
 			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/fr/services/environnement.html">Environnement et ressources naturelles</a></h3>
-				<p>Météo, climat, agriculture, faune, pollution, conservation, pêches</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/fr/services/defense.html">Sécurité nationale et défense</a></h3>
-				<p>Militaire, cybersécurité, sûreté des transports, sécuriser la frontière, contre-terrorisme</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/fr/services/culture.html">Culture, histoire et sport</a></h3>
-				<p>Arts, médias, patrimoine, langues officielles, identité nationale et financement</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/fr/services/police.html">Services de police, justice et urgences</a></h3>
-				<p>Sécurité, justice, se préparer en cas d'urgence, services aux victimes d'actes criminels</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/fr/services/transport.html">Transport et infrastructure</a></h3>
-				<p>Aviation, transport maritime, transport routier, transport ferroviaire, marchandises dangereuses, projets d'infrastructure</p>
+				<h3><a href="https://www.canada.ca/fr/services/sante.html">Santé</a></h3>
+				<p>Alimentation, nutrition, maladies, vaccins, médicaments, sécurité des produits et rappels</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://international.gc.ca/world-monde/index.aspx?lang=fra">Le Canada et le monde</a></h3>
-				<p>Politique étrangère, ententes commerciales, développement international, enjeux mondiaux</p>
+				<h3><a href="https://www.canada.ca/fr/services/environnement.html">Environnement et ressources naturelles</a></h3>
+				<p>Météo, climat, agriculture, faune, pollution, conservation, pêches</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
 				<h3><a href="https://www.canada.ca/fr/services/finance.html">Argent et finances</a></h3>
 				<p>Finances personnelles, dossiers de crédit, se protéger de la fraude, financer votre éducation</p>
 			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/fr/services/science.html">Science et innovation</a></h3>
-				<p>Recherche scientifique sur la santé, l'environnement et l'espace, subventions et financement</p>
-			</div>
+		</div>
+		<div>
+			<a class="btn btn-call-to-action" href="https://www.canada.ca/fr/services.html">Tous les services</a>
```

Comment:
HTML show the use of the newly added CSS style

```suggestion
			<a class="btn btn-all-services" href="https://www.canada.ca/fr/services.html">Tous les services</a>
```